npm2deb/utils.py

```diff
@@ -62,6 +62,8 @@ def get_template(filename):
 def get_watch(which):
     if which == 'github':
         return _templates.WATCH['github']
+    elif which == 'gitlab':
```

Comment:
I would prefer
```python
watch = _templates.WATCH
return watch.get(which, watch.get('npmregistry'))
```

Reply:
Could you do this change ? I'm not sure to understand

Reply:
sure.

Reply:
@shanavas786 : could you do it or show me what has to be done here ?